install/kubernetes/helm/istio/charts/tracing/templates/zipkin/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
My usual comment on helm-related PR - it would be great to get rid of this file, it creates a lot of confusion and upgrade problems.

The service names must be clear and not derived from the name of the chart.